doc/sphinx/source/getting_started/outputdata.inc

```diff
@@ -4,19 +4,162 @@
 Output
 ******
 
+ESMValTool automatically generates a new output directory with every run. The location is determined by 
+your config.yml's entry: output_dir variable, the recipe name, and 
+the date and time, using the the format: YYYYMMDD & HHMMSS.
+
+For instance, a typical output location would be: 
+output_directory/recipe_ocean_amoc_20190118_1027/
+
+This is effectively produced by the combination:
+$output_dir/recipe_name_YYYYMMDD_HHMMSS/
+
+
+
+This directory will contain 4 further subdirectories:
+
+1. preproc: This directory contains all the preprocessed netcdfs data & metadata.
+
+2. run: This directory includes all log files, a copy of the recipe, a summary of the resource usage, and the `settings.yml`_ and `metadata.yml`_ interface files. 
+
+3. work: A place for post processed files like NetCDF (depends on the diagnostics).
+
+4. plots: The location for all the plots, split by individual diagnostics and fields.
+
+
+
 Preprocessed datasets
 =====================
-preproc/
+
+The preprocessed datasets will be sent to the preproc/ directory.
+
+If the option save_intermediary_cubes is set to True in the config-user.yml file, then
+the intermediary cubes will also be saved here.
+
+
 
 Diagnostic output
 =================
-work/
+
+The work/ directory will contain all cubes that are output at the diagnostic stage.
+Ie, the model data is preprocessed by ESMValTool and stoed in the preproc/ directory.
+These files are opened by the diagnostic script, then some processing is applied.
+Once the diagnostic level processing has been applied, the resulting cubes should then 
+be saved to the work directory.
+
+It is possible to write your diagnostic such that it checks for the existance of 
+relevant processed cubes in the work directory. This means that re-running the
+diagnostic could be sped up.
+
+In python-based diagnostics, the work directory can be accessed with via the cfg dictionairy
+item, 'work_dir' ie: cfg['work_dir']
+
+In NCL-based diagnostics, this is given by config_user_info@work_dir.
+
+In NCL-based diagnostics, this is given by config_user_info@work_dir.
+
+
 
 Plots
 =====
-plots/
 
-Log files
-=========
-run/
-  
+The plots directory is where your diagnostics should save the output figures. 
+
+In python-based diagnostics, the plot directory can be accessed with via the cfg dictionairy
```

Comment:
This should not be part of a getting started document, but rather go in the diagnostic developers guidelines section

Reply:
This is defined in the new_diagnostic.inc file, so I'll delete it here.